scripts/update_graphql_e2e_test_protocol_version.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+# Copyright (c) Mysten Labs, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import re
+import argparse
+import difflib
+
+def replace_protocol_version_in_file(file_path, old_version, new_version, yes_to_all, dry_run):
+    with open(file_path, 'r') as file:
+        content = file.read()
+
+    updated_content = re.sub(
+        f"init --protocol-version {old_version} --",
+        f"init --protocol-version {new_version} --",
```

Comment:
This is a little bit brittle (as opposed to testing for lines containing `//# init` and then searching for `--protocol-version` somewhere in that line.

Reply:
Thanks for the review and suggestion @amnn ! Script updated to search for `//# init` first and then replace --protocol-version` with new version.